typings/mysql/index.d.ts

```diff
@@ -53,25 +53,51 @@ export function createPool(config: PoolOptions): BasePool;
 
 export function createPoolCluster(config?: PoolClusterOptions): PoolCluster;
 
-export function escape(value: any): string;
-
-export function escapeId(value: any): string;
-
+/**
+ * @file index.d.ts
+ * 
+ * @author marvinhagemeister
+ * @date 11 July 2017
+ * 
+ * @dependency \@types/sqlstring
+ * @see {@link https://www.npmjs.com/package/@types/sqlstring} for more information
+ */
+export function escape(value: any, stringifyObjects?: boolean, timeZone?: 'local' | NonNullable<unknown>): string;
```

Comment:
I reviewed the `sqlstring` code and the timezone always waits for a string, with two as default: `"local"` and `"Z"`.

I suppose the reason for choosing `NonNullable<unknown>` was to allow auto-suggestions.

`'local' | 'Z' | string & {}` seems more appropriate.

If `string & {}` sounds unfamiliar, we can use it like:

```ts
type CustomString = string & {};

type TimeZone = 'local' | 'Z' | CustomString;
```

Reply:
The NonNullable<unknown> was suggested by my IDE to be the perfect replacement for `string & {}`. Personally I don't really care which one's used. Oddly enough, the type of NonNullable is defined as `T & {}` yet using NonNullable<string> does not yield the same results as `string & {}`. 

I'll apply your change requests momentarily. Thank you for your assistance here.

Reply:
Is it alright with you if I'm keeping `string & NonNullable<string>` in, so the typescript-eslint rule "@typescript-eslint/ban-types" doesn't cause any errors?